src/openrct2/title/TitleScreen.cpp

```diff
@@ -225,6 +226,68 @@ void TitleScreen::TitleInitialise()
     {
         _sequencePlayer = GetContext()->GetUiContext()->GetTitleSequencePlayer();
     }
+    if (gConfigInterface.random_title_sequence)
+    {
+        
+        bool RCT1Installed = false, RCT1AAInstalled = false, RCT1LLInstalled = false;
+        int RCT1Count = 0, RCT1AACount = 0, RCT1LLCount = 0;
+        size_t scenarioCount = scenario_repository_get_count();
+        for (size_t s = 0; s < scenarioCount; s++ ) {
+            if (scenario_repository_get_by_index(s)->source_game == SCENARIO_SOURCE_RCT1) {
+                RCT1Count++;
+            }
+            if (scenario_repository_get_by_index(s)->source_game == SCENARIO_SOURCE_RCT1_AA) {
+                RCT1AACount++;
```

Comment:
Might as well just delete the `RCT1AACount` variable and use `RCT1AAInstalled` setting it to true as soon as there is a single park.